api/generator/generator.go

```diff
@@ -265,6 +272,14 @@ func (cg *CertGeneratorHandler) Handle(w http.ResponseWriter, r *http.Request) e
 		return errors.NewBadRequest(err)
 	}
 
+	if req.Format.Type == "pkcs12" {
+		var password []byte
+		if req.Format.Password != "" {
+			password = []byte(req.Format.Password)
+		}
+		pkcs12.ParseAndEncode(key, certBytes, password)
+	}
+
 	result := map[string]interface{}{
```

Comment:
This output should also contain the base64-encoded PKCS#12 file if it was requested.